client/Cargo.toml

```diff
@@ -23,8 +23,7 @@ serde_json = "~1.0.0"
 platforms = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu"]
 tier = "2"
 all-features = false
-exclude-crate-paths = [ { name = "*", exclude = "examples" },
-                        { name = "*", exclude = "test" },
+exclude-crate-paths = [ { name = "*", exclude = "test" },
```

Comment:
Removing it now because of the failing CI: https://github.com/canonical/hardware-api/actions/runs/16417390930/job/46386580848

Since the plan is to make the package non-native (see https://warthogs.atlassian.net/browse/HWAPI-38), I'll just patch the `clap` crate not to require examples dir to be present